bcmath/bcmath.php

```diff
@@ -94,7 +94,6 @@ function bcdiv ($dividend, $divisor, $scale = 0) {}
  * </p>
  * @return string|null the modulus as a string, or <b>NULL</b> if
  * <i>divisor</i> is 0.
- * @since 7.2 scale param added, dividend/divisor no longer truncated
```

Comment:
I suggest moving this information to the corresponding parameter blocks, but without `@`

Reply:
@slapoguzov Done!